platform/platform-filter-response/src/main/java/org/codice/ddf/platform/response/filter/ResponseFilter.java

```diff
@@ -29,29 +29,36 @@
 /**
  * Servlet filter that adds security information to the http response header.
  */
-public class ResponseSecurityFilter implements Filter {
+public class ResponseFilter implements Filter {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(ResponseSecurityFilter.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ResponseFilter.class);
 
     public static final String X_XSS_PROTECTION = "X-XSS-Protection";
 
     public static final String X_FRAME_OPTIONS = "X-Frame-Options";
 
     public static final String X_CONTENT_SECURITY_POLICY = "X-Content-Security-Policy";
 
+    public static final String CACHE_CONTROL = "Cache-Control";
+
     public static final String DEFAULT_XSS_PROTECTION_VALUE = "1; mode=block";
 
     public static final String DEFAULT_X_FRAME_OPTIONS_VALUE = "SAMEORIGIN";
 
     public static final String DEFAULT_CONTENT_SECURITY_POLICY =
             "default-src 'none'; connect-src 'self'; script-src 'self'; style-src 'self'; img-src 'self'";
 
+    public static final String DEFAULT_CACHE_CONTROL_VALUE =
+            "Cache-Control: \"public, max-age=604800, immutable\"";
```

Comment:
You mean different versions of DDF? I can't think of an easy way, though I'm open to suggestions. However, the different versions would have to be at the same location from the browser's perspective. I think that is unlikely to be a pain point outside of a dev environment, where we are used to clearing our cache anyways.